ncu: add page (#10617)
tldr-pages/tldr: pages/common/ncu.md

# ncu

> > Find newer versions of package dependencies and check outdated npm packages locally or globally.
> > More information: <https://github.com/raineorshine/npm-check-updates>.

Reviewer: ```suggestion
> Find newer versions of package dependencies and check outdated npm packages locally or globally.
> `ncu` only updates dependency versions in `package.json`. To install the new versions, run `npm install` afterwards.
> More information: <https://github.com/raineorshine/npm-check-updates>.
```

> 
> - Install npm package `node-check-updates` globally:
> 
> `npm install -g node-check-updates`

Reviewer: Is there any other page with an example of how to install the program?
```suggestion
```

Reviewer 2: Yep, We don't generally include installation commands to a page.

> - List outdated npm packages in the current directory:

Reviewer: ```suggestion
- List outdated dependencies in the current directory:
```

`ncu`

- List outdated global npm packages:

`ncu -g`

> - Upgrade all npm packages in the current directory:
> 
> `ncu -u && npm install`
> 
> - Interactively upgrade npm packages in the current directory:
> 
> `ncu -i && npm install`

Reviewer: ```suggestion
- Upgrade all dependencies in the current directory:

`ncu -u`

- Interactively upgrade dependencies in the current directory:

`ncu -i`
```

# ncu

> > npm plugin for `npm-check-updates`.

Reviewer 2: ```suggestion
> NPM plugin for `npm-check-updates`.
```

> > npm plugin for `npm-check-updates`.
> > Used for checking outdated npm packages locally or globally.

Reviewer 2: ```suggestion
> Find newer versions of package dependencies and check outdated npm packages locally or globally.
```
How about this?

> More information: <https://github.com/raineorshine/npm-check-updates>.

> - Install npm package node-check-updates globally:

Reviewer 2: ```suggestion
- Install npm package `node-check-updates` globally:
```

`npm install -g node-check-updates`

> - List outdated npm packages in current folder:

Reviewer 2: ```suggestion
- List outdated npm packages in the current directory:
```

`ncu`

- List outdated global npm packages:

`ncu -g`

> - Upgrade all npm packages in current folder:

Reviewer 2: ```suggestion
- Upgrade all npm packages in the current directory:
```

`ncu -u && npm install`

> - Interactively upgrade npm package in current directory:

Reviewer 2: ```suggestion
- Interactively upgrade npm packages in the current directory:
```

`ncu -i && npm install`

> - View `ncu` options:

Reviewer 2: ```suggestion
- Display help:
```
